docs/content/preview/migrate/performance.md

```diff
@@ -102,6 +94,14 @@ As more optimizations are introduced, average throughput increases. The followin
 
 {{< note title="Note" >}}
 
-With increased optimisations, the CPU usage also shoots up. Permitting the CPU usage beyond 60% may not be advisable as the database requires some cycles to do it's internal operations.
+- With increased optimisations, the CPU usage also shoots up. Permitting the CPU usage beyond 60% may not be advisable as the database requires some cycles to do it's internal operations.
+
+- These performance optimizations apply whether you are importing data using the yb-voyager [import data command](../migrate-steps/#import-data) or the [import data file command](../migrate-steps/#import-data-file).
 
 {{< /note >}}
+
+## Improve export performance
+
+By default, yb-voyager exports four tables at a time. To improve data export, parallelize the export of data from multiple tables using the [–-parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument with the export data command to increase the number of jobs. Setting the value too high can however negatively impact performance; a setting of '4' typically performs well.
```

Comment:
Not clear if yb-voyager is setting parallel-jobs to 4 by default, or what. number of jobs = number of tables exported in parallel?

Reply:
clarified the defaults.